.gitignore

```diff
@@ -15,6 +15,7 @@ dist/
 downloads/
 eggs/
 .eggs/
+.idea/
```

Comment:
could we remove the .idea dir from repo? thanks!

Reply:
Sure, Just remove it.